test/vm_tests/builder/codegen_test.dart

```diff
@@ -707,7 +707,7 @@ main() {
               (props) {
                 return Dom.div()();
               }, 
-              $BarPropsMixinConfig, // ignore: undefined_identifier
```

Comment:
This _seems_ like it should have been `$BarConfig` from the get go - my parser change use the name to detect that it's a generated config and therefore enforces the naming pattern `${FactoryName}Config`. Let me know if this doesn't work, though!